<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1264</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络嗅探器 Sniffer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="picture.qrc">
    <normaloff>:/new/pic/resource/sniffer.jpg</normaloff>:/new/pic/resource/sniffer.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="nalabel">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>10000</width>
      <height>10000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择网卡：</string>
    </property>
   </widget>
   <widget class="QLabel" name="filterlabel">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>10000</width>
      <height>10000</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>按协议过滤：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="naBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>711</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>10000</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="filterBox">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>30</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>10000</width>
      <height>10000</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>120</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始捕获！</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>120</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>暂停一下~</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>220</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&gt; 点我退出 &lt;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="allPktView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>801</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="binBrowser">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>581</width>
      <height>181</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QTreeWidget" name="pktDetails">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>1171</width>
      <height>181</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>包详情</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>530</y>
      <width>571</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="ip4label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>IPv4：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ip4lineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ip6lineEdit">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="ip6label">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>IPv6：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="arplineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="arplabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ARP：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tcplineEdit">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="tcplabel">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>60</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TCP：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="udplineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="udplabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>UDP：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="icmplineEdit">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>100</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="icmplabel">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>100</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ICMP：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="httplineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="httplabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>HTTP：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="otherlineEdit">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>140</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="otherlabel">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>140</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>其他：</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="binBrowser_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>571</width>
       <height>181</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>捕获的各协议包的数量</string>
     </property>
    </widget>
    <zorder>binBrowser_2</zorder>
    <zorder>ip4label</zorder>
    <zorder>ip4lineEdit</zorder>
    <zorder>ip6lineEdit</zorder>
    <zorder>ip6label</zorder>
    <zorder>arplineEdit</zorder>
    <zorder>arplabel</zorder>
    <zorder>tcplineEdit</zorder>
    <zorder>tcplabel</zorder>
    <zorder>udplineEdit</zorder>
    <zorder>udplabel</zorder>
    <zorder>icmplineEdit</zorder>
    <zorder>icmplabel</zorder>
    <zorder>httplineEdit</zorder>
    <zorder>httplabel</zorder>
    <zorder>otherlineEdit</zorder>
    <zorder>otherlabel</zorder>
   </widget>
   <widget class="QPushButton" name="aboutButton">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>220</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关于作者 ^o^</string>
    </property>
   </widget>
   <zorder>nalabel</zorder>
   <zorder>filterlabel</zorder>
   <zorder>naBox</zorder>
   <zorder>filterBox</zorder>
   <zorder>startButton</zorder>
   <zorder>stopButton</zorder>
   <zorder>exitButton</zorder>
   <zorder>allPktView</zorder>
   <zorder>binBrowser</zorder>
   <zorder>pktDetails</zorder>
   <zorder>aboutButton</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1264</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action111">
   <property name="text">
    <string>111</string>
   </property>
  </action>
  <action name="action111_2">
   <property name="text">
    <string>111</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1085</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>631</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
